README.md

```diff
@@ -98,4 +100,41 @@ We can extend the concept of the `as` syntax by generalizing the right hand side
 import value from "module" as {key1: "value1", key2: [1, 2, 3]};
 ```
 
-This would allow module attributes to scale up to multiple pieces of data, not privileging a single attribute.
+This would allow module attributes to scale to support a variety of metadata. It is currently unclear if the generalized constant form will be pursued in this proposal or in a follow up. In either case the addition of the generalized constant form is not hard to implement on top of the existing [spec text](https://tc39.github.io/proposal-module-attributes/). 
+
+## FAQ
+
+### Why not out of band?
+
+Why not both? The champions of this proposal think that exploring both an in- and out of band solutions to this problem are desirable. Depending on the environment in which JavaScript is being executed in either approach could be seen as desirable.
+
+While an in band solution is more verbose, it is also more straightforward for developers to adopt. For large project with many dependencies developers will not have to worry about creating a large manifest by compiling the meta data of all of their dependencies. Module authors will also not have to worry about shipping manifest in order for consumers to be able to run their modules.
+
+The [import-maps proposal][] is a great example of an out of band manifest that could be created at installation time, but it's focus is on modules and at this time all necessary information to generate an applications import map could be found in the package.json of it's dependencies.
+
+While there very well could be a future where an out of band manifest could be used for the purpose of identifying module type, the amount of infrastructure necessary for ecosystem adoption is significantly higher than the in band solution proposed here.
```

Comment:
given that the infrastructure required for in-band includes all the change any new syntax requires - eslint, babel, typescript, editors, browsers/engines - while out-of-band does not, can you elaborate a bit on how you're measuring "higher"?

Reply:
I think we could say things like, that identifying where to find the manifest is an extra piece of inherent complexity and ecosystem divergence: On the web, it may be a script tag; in Node.js, it may be package.json. Inline metadata eliminates this complexity in a comprehensive way. This forms an ecosystem adoption barrier; maybe that's what Myles is getting at.

I'd agree with @ljharb that "it's too much work to design this" is not a very strong argument (since we should probably do this anyway, e.g., for integrity) but I doubt that's what @MylesBorins was saying here.

Reply:
I've dropped the paragraph about infrastrucutre. my intent was that an out of band solution will require new forms of infrastructure and tooling that doesn't already exist. While new syntax does require updates to all of the tools you mentioned that is not particularly different than any other new syntax we add to the language. It also is an update that can happen under developers without requiring them to adopt any new tooling.

I'll think a bit more about how to best present this point, but for now I've dropped it as it seemed to not be having the desired impact